src/cc65/coptmisc.c

```diff
@@ -1091,3 +1142,54 @@ unsigned OptBinOps2 (CodeSeg* S)
     /* Return the number of changes made */
     return Changes;
 }
+
+
+unsigned OptTosPushPop(CodeSeg *S)
+/* Merge jsr pushax/j?? popax */
+{
+    unsigned Changes = 0;
+    unsigned I;
+
+    /* Walk over the entries */
+    I = 0;
+    while (I < CS_GetEntryCount (S)) {
+
+        const CodeEntry* N;
+
+        /* Get the next entry */
+        const CodeEntry* E = CS_GetEntry (S, I);
+
+        /* Check for decspn, incspn, subysp or addysp */
+        if (E->OPC == OP65_JSR                            &&
+            strcmp(E->Arg, "pushax") == 0                 &&
+            (N = CS_GetNextEntry (S, I)) != 0             &&
+            (N->OPC == OP65_JSR || N->OPC == OP65_JMP)    &&
+            strcmp(N->Arg, "popax") == 0                  &&
```

Comment:
```suggestion
            strcmp (N->Arg, "popax") == 0                 &&
```
There are more places like this.